datasafe-rest-impl/src/test/java/de/adorsys/datasafe/rest/impl/controller/DocumentControllerTest.java

```diff
@@ -19,70 +17,68 @@
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-public class DocumentControllerTest extends BaseDatasafeEndpointTest {
-
-    private static final String TEST_USER = "test";
-    private static final String TEST_PASS = "test";
-
+@Slf4j
+class DocumentControllerTest extends BaseTokenDatasafeEndpointTest {
 
     @MockBean
-    private DefaultDatasafeServices dataSafeService;
+    DefaultDatasafeServices dataSafeService;
```

Comment:
Maybe protected is better if it is intended to be used by subclasses

Reply:
ok